src/index.js

```diff
@@ -1,5 +1,5 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./components/app";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
```

Comment:
This line appears to be breaking when I run the app. Did you have the same issue? I see the file is named `app.jsx`?

Reply:
I updated app.jsx to App.jsx just because I am more used to that sort of naming convention for component files. I assume webpack didn't update it on my local server which is why I didn't get an error on my machine.